src/test/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfigTest.java

```diff
@@ -46,4 +47,20 @@ public void testSetHttpTimeoutsConfig() {
         (Integer) 15000
     );
   }
+
+  @Test
+  public void testSslConfigs() {
```

Comment:
should we add a test case for the default too? (which is `false`). Do we have a case for `https` in the URL? Or both these cases are covered from existing tests?

Reply:
Will address after resolving https://github.com/confluentinc/kafka-connect-elasticsearch/pull/278#discussion_r256499119